README.md

```diff
@@ -404,6 +404,41 @@ The recording will have tags as labels on the run.
 
 You can pass multiple tags using commas like `tag: node-10,nightly,staging`.
 
+### Specify auto cancel after failures
+
+Passes the number of failed runs to cancel a run after when using the [Cypress Cloud Auto Cancellation](https://docs.cypress.io/guides/cloud/smart-orchestration#Auto-Cancellation) feature.
+
+This feature requires Cypress 12.x or later and a [Cypress Cloud Business or Enterprise](https://www.cypress.io/cloud/) account.
+
+```yml
+name: Cypress E2E Tests
+on: [push]
+jobs:
+  cypress-run:
+    runs-on: ubuntu-22.04
+    name: E2E
+    steps:
+      - name: Setup Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 18
```

Comment:
Is node 18 required for this? `node-version` is technically optional, but recommended. I'm just thinking from a docs maintenance standpoint, maybe we consider using the a latest option to be more durable? https://github.com/actions/setup-node#supported-version-syntax